api/src/main/java/edu/cornell/mannlib/vitro/webapp/application/VitroHomeDirectory.java

```diff
@@ -210,4 +236,51 @@ public String toString() {
 		}
 	}
 
+    /**
+     * Populates home directory with home files, excluding the rdf directory
+     */
+    public void populate() {
+        File homeDestination = getPath().toFile();
+
+        if (!homeDestination.isDirectory() || homeDestination.list() == null) {
+            throw new RuntimeException("Application home dir is not a directory! " + homeDestination);
+        }
+        if (!homeDestination.canWrite()) {
+            throw new RuntimeException("Application home dir is not writable! " + homeDestination);
+        }
+        try {
+            copy(homeDestination);
+        } catch(Exception e) {
+            throw new RuntimeException("Failed to copy home files! " + homeDestination, e);
```

Comment:
Would be more appropriate to create a custom exception or a more specific one for improved error handling.

Reply:
Used more specific exception.